src/templates/app/_includes/pep_declaration.jsx

```diff
@@ -10,7 +10,7 @@ const PepDeclaration = () => (
         <div className='gr-padding-10 gr-12'>
             <input id='not_pep' type='checkbox' />
             <label htmlFor='not_pep'>
-                {it.L('I acknowledge that I am not a politically exposed person (PEP).')}
+                {it.L('I acknowledge that I am not a politically exposed person (PEP).')}{' '}
```

Comment:
why not add `&nbsp` instead of a literal empty space?

Reply:
i thought `{' '}` would be more consistent with jsx syntax, but sure we can use `&nbsp;` if that's preferred

Reply:
up to you, I guess :) there are more inconsistent things in our code than spacing convention.

Reply:
hmm, better to stick to one thing, i'll change it to `&nbsp;` :P